Plugins/Cirrious/Color/Cirrious.MvvmCross.Plugins.Color.WindowsCommon/Cirrious.MvvmCross.Plugins.Color.WindowsCommon.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This ToolsVersion change scares me a bit - but hopefully VS will ignore XS (let's wait and see :))